config/betterquesting/DefaultQuests/Quests/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==/StarsSoManyStars-QO1gudFvTtKgWe9942dW3Q==.json

```diff
@@ -0,0 +1,143 @@
+{
+  "questIDHigh:4": 4678501939106762450,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 3219
+    },
+    "1:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 3243
+    }
+  },
+  "questIDLow:4": -6892214430748682531,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "dreamcraft:item.MedalWarp",
+        "Count:3": 1,
+        "Damage:2": 0,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Stars, So Many Stars",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Eternal coils? Perfect time for stargate trophy number three. \nYou know the drill, you give 2 stargates\u0027 worth of stuff, you get your trophy. Simple as that.\n\n§3Our journey through time continues onwards, entering the UMV, aka DTPF era.\n\nSurprisingly, it still is the year 2022, but there have been a huge amount of changes since our last stop in the UEV era. The DTPF exists now, hypogen \u0026 eternal coils included, and machine parts up to UMV thus are craftable. Wireless energy is a thing noe too, both the LSC and the regular hatch kind, and AE2FC is in the pack, allowing direct fluid crafting (meaning no more volumetric flask shuffling). \nThe power situation hasnt changed much since the UEV era, except for dyson being even more bugged and giving free uncapped infinite power (abusing this was obviously labeled as bugabuse). \nSome other new useful things are the MABS, Integrated Ore Factory, Space Elevator, UIV and UMV singleblocks (and PAs accepting up to 64 UEV+ machines), EIG, EEC, industrial \u0026 mega apiary, yottanks, stocking buses, data bank chaining and the Draconic Fusion Crafter.\nThe stargate\u0027s cost has now skyrocketed far more than ever before, marking the beginning of this new era of GT:NH endgame, the era of expansion.\n\nThis time around the leading §aGDCloud§3 was challenged by the team §aNeo Caelestia§3, who were progressing at a staggering speed, inching ever closer to being first to gate, but ultimately weren\u0027t able to fully catch up and thus §aGDCloud§2§3 once again was first to conquer the challenge on the §a19th of November, 2022§3.\n\nP.S.: You know how this works, right? Here\u0027s the list:\n-PCB Factory, Component Assemblyline, AAL, Transcendent Plasma Mixer, EOH, QFT\n-Multi-Amp wireless hatches\n-Sub 1-tick OC, Batch Mode, inbuilt Void Protection on multis\n-Level Maintainer, Stocking Hatches, Crafting Input Buses\n\nP.P.S.: Chaotic capacitor bank? What\u0027s that?"
```

Comment:
```suggestion
      "desc:8": "Eternal coils? Perfect time for stargate trophy number three. \nYou know the drill, you give 2 stargates\u0027 worth of stuff, you get your trophy. Simple as that.\n\n§3Our journey through time continues onwards, entering the UMV, aka DTPF era.\n\nSurprisingly, it still is the year 2022, but there have been a huge amount of changes since our last stop in the UEV era. The DTPF exists now, hypogen \u0026 eternal coils included, and machine parts up to UMV thus are craftable. Wireless energy is a thing now too, both the LSC and the regular hatch kind, and AE2FC is in the pack, allowing direct fluid crafting (meaning no more volumetric flask shuffling). \nThe power situation hasnt changed much since the UEV era, except for dyson being even more bugged and giving free uncapped infinite power (abusing this was obviously labeled as bugabuse). \nSome other new useful things are the MABS, Integrated Ore Factory, Space Elevator, UIV and UMV singleblocks (and PAs accepting up to 64 UEV+ machines), EIG, EEC, industrial \u0026 mega apiary, yottanks, stocking buses, data bank chaining and the Draconic Fusion Crafter.\nThe stargate\u0027s cost has now skyrocketed far more than ever before, marking the beginning of this new era of GT:NH endgame, the era of expansion.\n\nThis time around the leading §aGDCloud§3 was challenged by the team §aNeo Caelestia§3, who were progressing at a staggering speed, inching ever closer to being first to gate, but ultimately weren\u0027t able to fully catch up and thus §aGDCloud§2§3 once again was first to conquer the challenge on the §a19th of November, 2022§3.\n\nP.S.: You know how this works, right? Here\u0027s the list:\n-PCB Factory, Component Assemblyline, AAL, Transcendent Plasma Mixer, EOH, QFT\n-Multi-Amp wireless hatches\n-Sub 1-tick OC, Batch Mode, inbuilt Void Protection on multis\n-Level Maintainer, Stocking Hatches, Crafting Input Buses\n\nP.P.S.: Chaotic capacitor bank? What\u0027s that?"
```

Reply:
Great, the diff view is not helpful here. It said `noe` instead of `now`.